### DISCUSSION TITLE
Plutot renderer cell documentation

### DISCUSSION THREAD

**Discussion by jeansebastienZ** [bosskmk/pluto_grid#134] [General]
Hello , 
Basically  my Cell Renderer not showing when all others kind of type ( text , date etc are showing ) , could you help me .

```
 PlutoColumn(
    title: 'column1',
    field: 'column1',
    type: PlutoColumnType.text(),
    enableEditingMode: false,
    renderer: (rendererContext) {
      return  Image.network(
          'https://flutter.dev/assets/flutter-lockup-1caf6476beed76adec3c477586da54de6b552b2f42108ec5bc68dc63bae2df75.png');

    },
  ),
```

  ```
plutorows.add(
            PlutoRow(
              cells: {
                'column1':PlutoCell(value : "testValue"), 
                 },
            ));
```
I think it's the PlutoCell value wich is the problem but i dont know how to init it . Is there a specific way to init renderer Cell ? The example using Faker package is not clear on this point  , moreover maybe it should be better to swith  PlutoColumnType.text() to PlutoColumnType.render() in the future   No ?

EDIT----

I found out , i couldnt use this image . 
I still point out the second point :  it should be better to swith  PlutoColumnType.text() to PlutoColumnType.render() in the future   what do you think of that  ?

Thanks you @bosskmk

**Comment by bosskmk** [comment]
First off, is your problem solved?

And I am thinking about render as follows.

Column types such as numeric type columns, date type columns, text type columns, etc. exist, and apart from them, the renderer is just a way of showing them.

I think PlutoColumnType.render is not a set of levels like PlutoColumnType.date, PlutoColumnType.text, PlutoColumnType.number.

**Comment by jeansebastienZ** [comment]
Yes , the problem was due to the Image link .
Ok , in term of choice , i understand .
In term of lisibilty , seeing PlutoColumnType.text , when the cell is a custom widget , i'm not sure it's really pertinent even if i understand your POV .

**Reply by bosskmk** [reply]
I think when there is a data field that contains the url of an image, its column type is text.

And when printing it, I think that it is possible to print the url as it is or change the url to an image and print it.

Also, when there is a data field in the form of numbers from 1 to 10.
I think that you can print this numerically or convert it to a progress bar and print it out.

From this point of view, I think the renderer should exist separately for each column.

But I understand your point of view.
I've focused on the source of the data, and you seem to be focusing on the result of the data being rendered.

Your comments like this are interesting.

My thoughts haven't changed from the ones mentioned above, but I'll take a cup of coffee and think again about your opinion.
